napari/layers/image/image.py

```diff
@@ -350,7 +350,7 @@ def rendering(self):
         * ``additive``: voxel colors are added along the view ray until
             the result is saturated.
         * ``iso``: isosurface. Cast a ray until a certain threshold is
-            encountered. At that location, lighning calculations are
+            encountered. At that location, lightning calculations are
```

Comment:
```suggestion
            encountered. At that location, lighting calculations are
```
Typo, but wrong fix.